Polished the GSoC contributors guide

emiliom opened the pull request:
Just stylistic tweaks to hopefully make the text slightly more user friendly. No new content.

leewujung commented on gsoc_contributors_guide.md:
```diff
@@ -43,74 +42,55 @@ of ocean sonar data.
+* Key data variables you will be working with:
+    * `Sv` (frequency, ping_time, range_bin) - volume backscattering strength measured from the echo
+    * `range` (frequency, ping_time, range_bin) - the measured range of an echo in meters 
+* Other variables included in this dataset so that the parameters used in the calibration (from raw to Sv) are recorded:
```
```suggestion
* Other variables included in this dataset. These are included so that the exact parameters used in the calibration (from raw to Sv) are recorded:
```

emiliom replied:
Done

leewujung reviewed (commented):
Thanks @emiliom , I like your style tweak!

One thing I think we should do something about is the first single file echogram. It is plotted with `range_bin` on the y-axis. I think this could cause confusion for contributors.

What do you think if we provide datasets that are already with coordinate `depth` (so we already combine the `range` and `water_level`), so that folks can folk on visualization instead of trying to figure out how the coordinates are mapped?

emiliom commented:
> One thing I think we should do something about is the first single file echogram. It is plotted with range_bin on the y-axis. I think this could cause confusion for contributors.

Yeah, it'd be nice. But is it worth it for this stage of things?? I added additional explanatory text (in addition to what I had already added) in the paragraph preceding the plot.

> What do you think if we provide datasets that are already with coordinate depth (so we already combine the range and water_level), so that folks can folk (work?) on visualization instead of trying to figure out how the coordinates are mapped?

It looks like the sample dataset that will be provided (via Valentina's notebook) is Hake Survey data. There, range_bin at least increases with depth (unlike the OOI echogram). For project proposals, that seems like enough (to me, anyway), together with **clear** statements about range_bin vs depth

leewujung commented:
I think: it probably will be ok, if we only provide the hake survey and OOI data as example datasets, because these two have uniform `sample_interval`, so `range_bin` and `range` are the same for all frequencies. It is the mapping that gets people confused.
